index.bs

```diff
@@ -4341,26 +4339,32 @@ or have any [=inherited interfaces=] that have [=interface members=] with these
 
     <blockquote>
 
-        To [=get the next iteration result=] for <code class="idl">SessionManager</code>, run the
-        following steps:
+        The [=async iterator initialization steps=] for a <code class="idl">SessionManager</code>
+        async iterator |iterator| are:
+
+        1. Set |iterator|'s current state to "not yet started".
```

Comment:
Should "current state" be xreffed like https://heycam.github.io/webidl/#environment-ready-promise ?

Reply:
Yeah, that's probably a good idea; I'll push that tweak tomorrow before merging.